app/src/main/java/org/mozilla/rocket/privately/browse/BrowserFragment.kt

```diff
@@ -178,16 +176,6 @@ class BrowserFragment : LocaleAwareFragment(),
         return false
     }
 
-    fun setIsLoadingListener(listener: LoadStateListener) {
```

Comment:
Remove the now unused LoadStateListener in BrowserFragment.java, too?

Reply:
It's been there since day 1 (@ahunt  if I remember correctly.) I'd hesitate to remove old code unless it's required. The one added in BrowserFragment.kt is added by @josephyu1110 recently. So I'm confident we can remove it here.

For the LoadStateListener in BrowserFragement, I'll remove it when we do Session refactoring. Since this PR is for IdelingResource refinement, I'll do it separately.